doc/TokenCollector.md

```diff
@@ -1,6 +1,8 @@
 # TokenCollector
 
-In Tokenlon V6, multiple schemes of token approval is supported. TokenCollector is an abstract contract that handles different ways of token transfering. When interacting with Tokenlon, user can choose one of supported approving scheme and prepare the corresponded Tokenlon permit parameter (in bytes). The first byte of the permit indicate the type and the rest are encoded data with type specific structure.
+`TokenCollector` is an abstract contract designed to handle various token collection mechanisms. It supports different methods of token transfer, allowing flexibility in interacting with Tokenlon and other token standards.
```

Comment:
> `... allowing flexibility in interacting with Tokenlon and other token standards`

What does this mean?

Reply:
This means:

- Tokenlon uses TokenCollector to collect user's tokens.

- It supports multiple schemes to interact with various token types.

This flexibility allows TokenCollector to handle different token standards and transfer methods efficiently.

Reply:
It simply describes that we actually define `TokenCollector` as an abstract contract.